src/examples/learn_quant/README.md

```diff
@@ -0,0 +1,35 @@
+# Learning quantifier expressions and measuring monotonicity
+
+This example is under ongoing development.
+
+The code in this example generates models of quantifier expressions for testing their learnability. For a greater understanding of the different components of this example, follow the (tutorial)[tutorial.ipynb].
+
+## Contents
+
+- `scripts`: a set of scripts for generating `QuantifierModels` and measuring various properties of individual models and sets of models.  These are explained in more detail in the [Usage](#usage) section below.
+- `outputs`: outputs from the generation routines for creating `QuantifierModel`s and `QuantifierUniverse`s
+- `referents.csv`: this file defines the set of points of communication (which will become `Referent`s in ULTK terms).
+- `meaning.py`: this file defines the meaning space (a `Universe` in ULTK terms) of referents that are individual models of quantifiers (`QuantifierModel`s)
```

Comment:
very small terminological thing: they aren't models _of_ quantifiers, but more like models "for" quantifiers.  In Generalized Quantifier Theory, a quantifier is a _set of models_ in the sense of `QuantifierModel`